lib/tasks/activate.rake

```diff
@@ -0,0 +1,7 @@
+namespace :deploy do
+  desc 'Run activate script'
+  task activate: :environment do
```

Comment:
rubocop might complain.... but I think this script *does not* need the env, hence why it was able to run as a standalone script, which would make it faster if we wanted it to be

Reply:
actually I think this script should probably run outside the environment
```suggestion
  # rubocop:disable [Rails/EnvironmentRakeTask] or something
  task :activate do
  # rubocop:enable [Rails/EnvironmentRakeTask] or something
```

Reply:
Good call, it doesn't!